client/my-sites/backup/controller.js

```diff
@@ -105,7 +107,32 @@ export function showUnavailableForMultisites( context, next ) {
 
 	const state = context.store.getState();
 	const siteId = getSelectedSiteId( state );
-	if ( isJetpackSiteMultiSite( state, siteId ) ) {
+	const sitePlan = getSitePlan( state, siteId );
+	const siteProducts = getSiteProducts( state, siteId );
+
+	const siteHasBackupPlan = () => {
```

Comment:
This should definitely be a selector!

Reply:
Done.

Reply:
I'm creating a unit test for the new selector..